src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecoratorTest.java

```diff
@@ -376,7 +376,7 @@ private ProcReturn execCommandInContainer(String containerName, Node node, boole
             Thread.sleep(100);
         }
         assertFalse("proc is alive", proc.isAlive());
-        int exitCode = proc.joinWithTimeout(10, TimeUnit.SECONDS, StreamTaskListener.fromStderr());
+        int exitCode = proc.joinWithTimeout(20, TimeUnit.SECONDS, StreamTaskListener.fromStderr());
```

Comment:
We already have

```java
@Test(timeout = 10000)
```

so

* Does this even help?
* Would it really be necessary?

Suggest

```suggestion
        int exitCode = proc.join();
```

and for the test overall

```java
@Test(timeout = 180_000)
```

matching `JenkinsRule.timeout`.

Reply:
Since this is used across multiple tests, I used a global timeout